registry/registry.go

```diff
@@ -17,6 +17,9 @@ import (
 	"github.com/docker/go-connections/sockets"
 	"github.com/docker/go-connections/tlsconfig"
 	"github.com/sirupsen/logrus"
+
+	"github.com/docker/docker/pkg/homedir"
```

Comment:
nit: looks like gofmt fix-up was done in https://github.com/moby/moby/pull/40349, so possibly pick that one as well

Reply:
^ didn't apply cleanly, so not worth spending too much time on